src/backend/converter/olr_event_handler.c

```diff
@@ -733,13 +783,28 @@ parseOLRDDL(Jsonb * jb, Jsonb * payload, orascn * scn, orascn * c_scn, orascn *
 						elog(DEBUG1, "  ADD COLUMN: %s", col->colname);
 						ddlcol->name = pstrdup(col->colname);
 
+						if (synchdb_letter_casing_strategy == LCS_AS_IS)
+						{
+							if (isallupper(ddlcol->name))	/* need to normalize to lowercase */
```

Comment:
Does LCS_AS_IS mean “as it is” when you’re converting the value inside? I’m confused. 
Same comments to the LCS_AS_IS uses below.